lldb/source/Plugins/ExpressionParser/Clang/ClangUserExpression.cpp

```diff
@@ -695,9 +689,8 @@ bool ClangUserExpression::Parse(DiagnosticManager &diagnostic_manager,
       // so recreate those.
       CreateSourceCode(retry_manager, exe_ctx, m_imported_cpp_modules,
                        /*for_completion*/ false);
-      parse_success = TryParse(retry_manager, exe_scope, exe_ctx,
-                               execution_policy, keep_result_in_memory,
-                               generate_debug_info);
+      parse_success = TryParse(retry_manager, exe_ctx, execution_policy,
+                               keep_result_in_memory, generate_debug_info);
```

Comment:
Per initial comments above this would become:
```
parse_success = TryParse(retry_manager, exe_ctx.GetBestExecutionContextScope(), execution_policy,
                               keep_result_in_memory, generate_debug_info);
```